components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html

```diff
@@ -660,18 +660,11 @@
   /* ]]> */
 </script>
     <div id="channel-window-help" style="display: none;">
-{% comment %}
-      <h2>Setting the Default Image Display Settings:</h2>
-      <p>You can use the "Rendering Details" dialog box to change the default settings for viewing this image.</p>
-      <p>To change the color displayed for a channel, click on the color box at the right, which opens the "Choose Color" dialog box. You can choose the color using the interactive features at the top. Hex color codes can be input in the text box. Preset colors can be selected at the bottom. Changes take effect when you click "Apply".</p>
-      <p>To change the color intensity, drag the slider buttons for the individual channels to the left or right. This changes the values that are used for the minimum and maximum displays settings. Changes take effect when you click "Apply".</p>
-      <p>Once the image is displayed correctly, click "Save Settings". The image thumbnail, the image in the "Mini Viewer", and the image in the "Full Viewer" will now use these display settings. Clicking the "Reset" button will reset the display settings to the most recently saved settings.</p>
-      <p>Viewers will be able to adjust display settings for viewing images, but not save them.</p>
-      {% endcomment %}
       <h2>Changing the Image Display Settings:</h2>
-      <p>You can use the "Rendering Details" dialog box to change the settings for viewing this image.</p>
-      <p>To change the color displayed for a channel, click on the color box at the right, which opens the "Choose Color" dialog box. You can choose the color using the interactive features at the top. Hex color codes can be input in the text box.  Preset colors can be selected at the bottom. Changes take effect when you click "Apply".</p>
-      <p>To change the color intensity, drag the slider buttons for the individual channels to the left or right. This changes the values that are used for the minimum and maximum displays settings. Changes take effect when you click "Apply".</p>
+      <p>You can use the "Rendering Details" dialog box to change the settings for viewing this image.
+        The changes will not be saved until you click the "Save" button.</p>
+      <p>To change the color displayed for a channel, click on the color-picker icon at the right, which opens the "Choose Color" dialog box. You can choose the color using the interactive features at the top. Hex color codes can be input in the text box.  Preset colors can be selected at the bottom. Changes take effect when you click "OK".</p>
+      <p>To change the color intensity, drag the slider handles for the individual channels to the left or right. This changes the values that are used for the minimum and maximum displays settings.</p>
```

Comment:
Maybe make it shorter, and more imperative by removing some of the "you can" and double negatives from the text and clearing up some of the wording to be simpler.  For example:

```
Use the "Rendering Details" dialog box to change the viewer settings for this image.  The changes will be saved when you click the "Save" button.

To change the channel color, click on the color-picker icon at the right to open the "Choose Color" dialog box. Using the color using the selector at the top, select the desired hue using the outer ring, and saturation and value with the inner square. Alternatively, hexadecimal color codes may be input in the text box, or preset colors may be selected at the bottom. The changes will take effect when you click "OK".

To change the color intensity, drag the slider handles for the individual channels to the left or right to change the the minimum and maximum display values.</p>
```